lte/gateway/python/magma/pipelined/tests/test_enforcement.py

```diff
@@ -400,6 +431,45 @@ def test_two_subscribers(self):
 
         flow_verifier.verify()
 
+    def test_subscriber_policy_with_he(self):
+        """
+        Add policy to subscriber, send 4096 packets
+
+        """
```

Comment:
Can you just create a test_he.py which will have the he rules? I feel like expanding enforcement test is becoming too cubersome.

Reply:
that way we don't need to change the existing enforcement snapshots